hedera-node/hedera-app/src/main/java/com/hedera/node/app/tss/handlers/TssShareSignatureHandler.java

```diff
@@ -131,11 +128,10 @@ private void validateAndAccumulateSignatures(
             final Bytes messageHash,
             final long shareIndex,
```

Comment:
We won't ever have a `long` # of shares.

Reply:
Yeah it was defined long in protobuf. I can modify in next PR.